Source/include/Ably/ARTSummaryTypes.h

```diff
@@ -0,0 +1,193 @@
+ */
++ (nullable instancetype)createFromDictionary:(NSDictionary *)dictionary;
+
+/**
+ * Writes the summary type data to a mutable dictionary.
+ * @param dictionary The dictionary to write to
+ */
+- (void)writeToDictionary:(NSMutableDictionary<NSString *, id> *)dictionary;
+
+@end
+
+/**
+ * TM7c1: Summary with total count and list of client IDs.
+ */
+NS_SWIFT_SENDABLE
+@interface ARTSummaryClientIdList : NSObject <ARTDictionarySerializable>
+
+/// Total count of items
+@property (nonatomic, readonly) NSInteger total;
+
+/// Array of client IDs
+@property (nonatomic, readonly, copy) NSArray<NSString *> *clientIds;
+
+/// Whether the summary data is clipped (truncated)
+@property (nonatomic, readonly) BOOL clipped;
+
+/**
+ * Initializes with total count and client IDs array.
+ * @param total The total count
+ * @param clientIds Array of client ID strings
+ */
+- (instancetype)initWithTotal:(NSInteger)total clientIds:(NSArray<NSString *> *)clientIds;
+
+/**
+ * Initializes with all properties.
+ * @param total The total count
+ * @param clientIds Array of client ID strings
+ * @param clipped Whether the data is clipped
+ */
+- (instancetype)initWithTotal:(NSInteger)total 
+                    clientIds:(NSArray<NSString *> *)clientIds
+                      clipped:(BOOL)clipped;
+
+@end
+
+/**
+ * TM7d1: Summary with total count and dictionary mapping client IDs to counts.
+ */
+NS_SWIFT_SENDABLE
+@interface ARTSummaryClientIdCounts : NSObject <ARTDictionarySerializable>
+
+/// Total count of items
+@property (nonatomic, readonly) NSInteger total;
+
+/// Dictionary mapping client IDs to their counts
+@property (nonatomic, readonly, copy) NSDictionary<NSString *, NSNumber *> *clientIds;
+
+/// Whether the summary data is clipped (truncated)
+@property (nonatomic, readonly) BOOL clipped;
+
+/// Total number of unidentified annotations
+@property (nonatomic, readonly) NSInteger totalUnidentified;
+
+/// Total number of unique client IDs
+@property (nonatomic, readonly) NSInteger totalClientIds;
+
+/**
+ * Initializes with total count and client ID counts dictionary.
+ * @param total The total count
+ * @param clientIds Dictionary mapping client IDs to their counts
+ */
+- (instancetype)initWithTotal:(NSInteger)total clientIds:(NSDictionary<NSString *, NSNumber *> *)clientIds;
+
+/**
+ * Initializes with all properties.
+ * @param total The total count
+ * @param clientIds Dictionary mapping client IDs to their counts
+ * @param clipped Whether the data is clipped
+ * @param totalUnidentified Total unidentified annotations
+ * @param totalClientIds Total unique client IDs
+ */
+- (instancetype)initWithTotal:(NSInteger)total 
+                    clientIds:(NSDictionary<NSString *, NSNumber *> *)clientIds
+                      clipped:(BOOL)clipped
+            totalUnidentified:(NSInteger)totalUnidentified
+               totalClientIds:(NSInteger)totalClientIds;
+
+@end
+
+/**
+ * TM7e1: Summary with only total count.
```

Comment:
Why do we have public doc comments that refer to spec points? That means nothing for a user.

Furthermore there are already proper documentation comments for these types in the spec repo.

Reply:
https://github.com/ably/ably-cocoa/pull/2080/commits/3bb5a3f61db86fe0cb6c92cfcce1d6431c159bc6